src/server/_query.py

```diff
@@ -174,29 +174,25 @@ def filter_pair(pair: SourceSignalPair, i) -> str:
 def filter_time_pairs(
```

Comment:
```suggestion
def filter_time_pair(
```

and then propagate this renaming out to `where_time_pairs` -> `where_time_pair` and update wherever that's called?